local/config/wp-tests-config.php

```diff
@@ -3,6 +3,7 @@
  * Config file used only for phpunit tests inside Docker.
  *
  * phpcs:disable WordPress.WP.GlobalVariablesOverride.Prohibited
+ * phpcs:disable Squiz.Commenting.FileComment.MissingPackageTag
```

Comment:
Why not just add the package name?

Reply:
Because this is a `wp-config.php` file and won't live within the plugin during use. :man_shrugging: